.gitattributes

```diff
@@ -0,0 +1,11 @@
+# Set default behavior to automatically normalize line endings.
+* text=auto
+
+# Force batch scripts to always use CRLF line endings so that if a repo is accessed
+# in Windows via a file share from Linux, the scripts will work.
+*.{cmd,[cC][mM][dD]} text eol=crlf
+*.{bat,[bB][aA][tT]} text eol=crlf
```

Comment:
I wonder if we should add `.ps1` ([PowerShell](https://learn.microsoft.com/en-us/powershell/)) here?

Reply:
right, will do